lib/selector/tokenizer/tokenizer.go

```diff
@@ -54,7 +54,7 @@ type Token struct {
 }
 
 const (
-	identifierExpr = `[a-zA-Z_./-][a-zA-Z0-9_./-]*`
+	identifierExpr = `[a-zA-Z0-9]?([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])*`
```

Comment:
> or does the subdomain prefix never get passed to us?

We don't do anything with the labels provided to k8s.  As in, we don't modify them..  they come to us unchanged.